src/elements/picker/extras/entity-picker-results/EntityPickerResults.ts

```diff
@@ -65,10 +66,12 @@ import { NovoLabelService } from '../../../../services/novo-label-service';
                     </p>
                 </item-content>
             </novo-list-item>
+            <novo-loading theme="line" *ngIf="isLoading && matches.length > 0"></novo-loading>            
         </novo-list>
     `,
     host: {
-        '[hidden]': 'matches.length === 0'
+        '[hidden]': 'matches.length === 0',
+        '(scroll)': 'onScrollDown($event)'
```

Comment:
Can you move this to `@HostListener`? These will be deprecated eventually.

https://github.com/angular/angular/blob/4.0.0-beta.2/modules/%40angular/core/src/metadata/directives.ts#L981